js-exercises/freq-sort/freqSort.js

```diff
@@ -0,0 +1,25 @@
+function freqSort(arrayOfCharacters) {
+  if (!Array.isArray(arrayOfCharacters)) {
+    throw new Error("argument should be array");
+  } else {
+    let frequencyMap = {};
+
+    arrayOfCharacters.map(checkFreqElement);
+
+    function checkFreqElement(element) {
+      element = element.normalize("NFC");
+      if (frequencyMap.hasOwnProperty(element))
+        return (frequencyMap[element] += 1);
```

Comment:
Here instead of using frequencyMap.hasOwnProperty try to use Object.prototype.hasOwnProperty bcz if frequency has a key as 'hasOwnProperty' things will not work as expected, its a very rare thing but its good to do like this.